doc/CMakeLists.txt

```diff
@@ -9,14 +9,14 @@ IF (CMAKE_VERSION VERSION_LESS "3.14")
 		MESSAGE("Hard coded doc destination")
 		INSTALL(FILES AUTHORS DESTINATION "${CMAKE_INSTALL_DOCDIR}"  RENAME AUTHORS.SF)
 		INSTALL(FILES "${PROJECT_SOURCE_DIR}/../AUTHORS" DESTINATION "${CMAKE_INSTALL_DOCDIR}" RENAME AUTHORS.GITHUB)
-		INSTALL(FILES "${PROJECT_SOURCE_DIR}/../ChangeLog" DESTINATION "${CMAKE_INSTALL_DOCDIR}")
+		INSTALL(FILES ChangeLog DESTINATION "${CMAKE_INSTALL_DOCDIR}")
 	ENDIF (UNIX)
 ELSE (CMAKE_VERSION VERSION_LESS "3.14")
 	# This is the preferred instruction, but it's not available until cmake 3.14
 	MESSAGE("Newer CMake version, using preferred method")
 	INSTALL(FILES AUTHORS TYPE DOC RENAME AUTHORS.SF)
 	INSTALL(FILES "${PROJECT_SOURCE_DIR}/../AUTHORS" TYPE DOC RENAME AUTHORS.GITHUB)
-	INSTALL(FILES "${PROJECT_SOURCE_DIR}/../ChangeLog" TYPE DOC)
+	INSTALL(FILES ChangeLog TYPE DOC)
```

Comment:
ahh...yeah - my bad there. Good catch.

I had a `ChangeLog` in the project root which was the git log dump; missed that it wasn't checked in.

Any how...we should reformat this file to match up, or may be just entirely re-dump it using your suggestion since the whole SVN history got migrated to git; but that can be another PR.

Reply:
Not a worry. Thanks for the speedy resolution, I'm building now to test, as well as getting some play testing in as well.